docs/application-flow.rst

```diff
@@ -390,17 +389,130 @@ With the run of each sector involving the following steps:
+
+The final step of preparing the investment process is to compute the constrains, e.g. factors that will determine how much a technology could be invested in and include things like matching the demand, the search rules calculated above, the maximum production of a technology for a given capacity or the maximum capacity expansion for a given time period. Available constrains are set in the subsector section of the ``settings.toml`` file and described in :py:mod:`muse.constrains`. By default, all of them are applied. Note that these constrains might result in unfeasible situations if they do not allow the production to grow enough to match the demand. This is one of the common reasons for a MUSE simulation not converging.
+
+With all this information, the investment process can proceed. This is done per sector using the method described by the ``lpsolver`` in the ``settings.toml`` file. Available solvers are described in :py:mod:`muse.investments`
+
+If the investment succeeds, the new installed capacity will become part of the agents' assets.
+
+
+Dispatch
+~~~~~~~~
+
+The dispatch stage when running a sector can be described by the following graph:
+
+.. graphviz::
+    :align: center
+    :alt: Dispatch stage of a sector calculation
+
+    digraph dispatch {
+            fontname="Helvetica,Arial,sans-serif"
+            node [fontname="Helvetica,Arial,sans-serif", shape=box, style=rounded]
+            edge [fontname="Helvetica,Arial,sans-serif"]
+            rankdir=LR
+            clusterrank=local
+            newrank=true
+
+            {node [shape=""]; start; end;}
+            capacity [label="Aggregate capacity\nfrom all agents"]
+            supply [label="Calculate supply"];
+            consumption [label="Calculate consumption"];
+            cost [label="Calculate cost"];
+            market [label="Create sector\nmarket"]
+            dispatch[label="dispatch_production\n(settings.toml)", fillcolor="#ffb3b3", style="rounded,filled"]
+
+
+            start ->  capacity -> supply -> consumption -> cost -> market -> end
+            dispatch -> supply
+        }
+
+After the investment stage is completed, then the new capacity of the sector is obtained by aggregating the assets of all agents of the sector. Then, the supply of commodities is calculated as requested by the ``dispatch_production`` argument defined for each sector in the ``settings.toml`` file.
+
+The typical choice used in most examples in MUSE is ``share``, where the utilization across similar assets is the same in percentage. However, there are other options available, like
+
+- ``costed``: assets are ranked by cost and the cheaper ones are allowed to service the demand first up to their maximum production.
```

Comment:
The assets are ranked by their levelised costs.
Minimum service can be imposed if present.